docs/nodes/archive-node/docker.md

```diff
@@ -150,15 +149,15 @@ eg.
 docker logs -f -n 100 $(docker ps -aq --filter name="astar-container")
 ```
 
-### Indexers and oracles
+### Indexers and Oracles
 
-To access data from indexers (lke The Graph) or Oracles (like Chainlink), you need to add the debug flags below to the node launch command, after the `astar-collator` line:
+To access data from indexers (like The Graph) or Oracles (like Chainlink), add the follwing debug flags to the node launch command, after the `astar-collator` line:
 
 `--ethapi=debug`
 
-### Upgrade node
+## Upgrade node / Update node
```

Comment:
is it not either or?

Reply:
whichever one is more appropriate, as long as both 'update node' and 'upgrade node' are present in the blurb for the AI to notice.